lisa/schema.py

```diff
@@ -178,6 +178,33 @@ class TypedSchema:
     type: str = ""
```

Comment:
Trying to review commit message here: Anything more concrete to put as output example? Still vague :/

Reply:
The commit message is not a good place to introduce functionality. will provide more details in following document updates.

Reply:
Why not? It's the front door for devs to grasp it. Not meaning a lot of pages, just the bare minimum to explain what is going on.